core/src/main/scala/cromwell/core/logging/WorkflowLogger.scala

```diff
@@ -18,8 +18,14 @@ import scala.util.Try
 
 trait WorkflowLogging extends ActorLogging { this: Actor =>
   def workflowIdForLogging: WorkflowId
-
-  lazy val workflowLogger = new WorkflowLogger(self.path.name, workflowIdForLogging, Option(log))
+  def rootWorkflowIdForLogging: WorkflowId
+
+  lazy val workflowLogger = new WorkflowLogger(
+    self.path.name,
+    workflowIdForLogging,
+    rootWorkflowIdForLogging,
```

Comment:
TOL use keyword args to make sure the consecutive same-typed workflow ID params are being passed correctly?